src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/DefaultHtmlGeneratorExtensions.cs

```diff
@@ -47,5 +48,25 @@ public static class DefaultHtmlGeneratorExtensions
 
             return tagBuilder;
         }
+
+        public static TagBuilder GeneratePageForm(
```

Comment:
Shouldn't have an extension method. We only did fragments this way for other generator bits because it was added later.